gliderpy/plotting.py

```diff
@@ -90,3 +90,44 @@ def plot_transect(
     ax.set_ylim(ax.get_ylim()[0], 0)
 
     return fig, ax
+
+
+@register_dataframe_method
+def plot_ctd(
+    df: pd.DataFrame,
+    profile_number: int,
+    var: str,
+    ax: plt.Axes = None,
+    color: str | None = None,
+) -> tuple:
+    """Make a CTD profile plot of pressure vs property
+    depending on what variable was chosen.
+
+    :param profile_number: profile number of CTD
+    :param var: variable to plot against pressure
+    :param ax: existing axis to plot on (default: None)
+    :param color: color for the plot line (default: None)
+    :return: figure, axes
+    """
+    g = df.groupby(["longitude", "latitude"])
+    profile = g.get_group(list(g.groups)[profile_number])
+
+    if ax is None:
+        fig, ax1 = plt.subplots(figsize=(5, 6))
+        ax1.plot(profile[var], -profile["pressure"], label=var, color=color)
```

Comment:
Pressure should be a positive number. We can invert the axis here instead with something like:

```python
    if not ax.yaxis_inverted():
        ax.invert_yaxis()
```

BTW, I'm confused with ax, ax1, ax2. Aren't we plotting just a single variable on `ax`?

```suggestion
        ax1.plot(profile[var], profile["pressure"], label=var, color=color)
```

Reply:
We are using the command twiny for duplicating axes. I couldn't solve it just using "ax" for both, so I decided to specify.